test/e2e/networks/single_secp.toml

```diff
@@ -0,0 +1,3 @@
+key_type = "secp256k1"
```

Comment:
I'm guessing this was just used while testing, I think we can just drop committing these since they are trivial to set up again if needed.